fabric-chaincode-shim/build.gradle

```diff
@@ -38,7 +38,7 @@ checkstyleTest {
 
 
 
-check.dependsOn dependencyCheckAnalyze
+// check.dependsOn dependencyCheckAnalyze
```

Comment:
You should be able to skip this step using `-x dependencyCheckAnalyze` when building locally if required